ui/raidboss/data/03-hw/alliance/dun_scaith.js

```diff
@@ -523,27 +474,21 @@
       regexJa: Regexes.startsUsing({ id: ['1C0E', '1C20'], capture: false }),
       regexCn: Regexes.startsUsing({ id: ['1C0E', '1C20'], capture: false }),
       regexKo: Regexes.startsUsing({ id: ['1C0E', '1C20'], capture: false }),
-      alertText: {
-        en: 'Look away',
-        fr: 'Regardez ailleurs',
-      },
+      response: Responses.lookAway(),
     },
     {
       id: 'Dun Scaith Noctoshield',
-      regex: Regexes.gainsEffect({ target: 'Diabolos', effect: 'Noctoshield', capture: false }),
-      regexDe: Regexes.gainsEffect({ target: 'Diabolos', effect: 'Nachtschild', capture: false }),
-      regexFr: Regexes.gainsEffect({ target: 'Diabolos', effect: 'Nocto-Bouclier', capture: false }),
-      regexJa: Regexes.gainsEffect({ target: 'ディアボロス', effect: 'ノクトシールド', capture: false }),
-      regexCn: Regexes.gainsEffect({ target: '迪亚波罗斯', effect: '夜障', capture: false }),
-      regexKo: Regexes.gainsEffect({ target: '디아볼로스', effect: '밤의 방패', capture: false }),
+      regex: Regexes.gainsEffect({ target: 'Diabolos', effect: 'Noctoshield' }),
+      regexDe: Regexes.gainsEffect({ target: 'Diabolos', effect: 'Nachtschild' }),
+      regexFr: Regexes.gainsEffect({ target: 'Diabolos', effect: 'Nocto-Bouclier' }),
+      regexJa: Regexes.gainsEffect({ target: 'ディアボロス', effect: 'ノクトシールド' }),
+      regexCn: Regexes.gainsEffect({ target: '迪亚波罗斯', effect: '夜障' }),
+      regexKo: Regexes.gainsEffect({ target: '디아볼로스', effect: '밤의 방패' }),
       suppressSeconds: 5,
       condition: function(data) {
         return data.role == 'tank' || data.role == 'healer';
       },
-      alertText: {
-        en: 'Boss hitting hard--Shield/Mitigate',
-        fr: 'Le boss frappe fort - Bouclier/Mitigation',
-      },
+      response: Responses.tankBuster(),
```

Comment:
This is different.  It's not really a tankbuster, it's more like hard autos?

Reply:
would be miniBuster here ok? or should i revert it?

Reply:
I think the difference here is that this is over a longer period of time, so a buster call on its own might make somebody think it was fine and then over? I'm not sure.  Maybe revert for now and can revisit later.

Reply:
ok it is now reverted

Reply:
ok it is now reverted